airflow/models/baseoperator.py

```diff
@@ -1001,6 +1005,7 @@ def __init__(
         self.doc_yaml = doc_yaml
         self.doc_rst = doc_rst
         self.doc = doc
+        self._task_display_name = task_display_name
```

Comment:
Instead of trying to store the original value (making `_task_display_name` and `task_display_name` different values, which is confusing), it’s probably easier to just do

```python
self.task_display_name = task_display_name or task_id
```

Reply:
Same for `dag_display_name`.

Reply:
Okay, also here existing code I took over from the previous PR. In my view actually mapping the private name to the property was obvious a good choice. But if you have a different opinion, also fine with this. Renamed:
- `_task_display_name` --> `_task_display_property_value`
- `_dag_display_name` --> `_dag_display_property_value`